fbw-a380x/src/systems/instruments/src/EWD/elements/ThrustGauge.tsx

```diff
@@ -96,26 +96,27 @@ export class ThrustGauge extends DisplayComponent<ThrustGaugeProps> {
 
```

Comment:
Github won't let me comment up past here, but you should delete thrustLimitMax as it's no longer used.